platform-service/src/unstract/platform_service/helper/adapter_instance.py

```diff
@@ -28,21 +28,23 @@ def get_adapter_instance_from_db(
         """
         if check_feature_flag_status(FeatureFlag.MULTI_TENANCY_V2):
             query = (
-                "SELECT id, adapter_id, adapter_metadata_b FROM "
-                f'"{DB_SCHEMA}".{DBTableV2.ADAPTER_INSTANCE} x '
+                "SELECT id, adapter_id, adapter_name, adapter_type, adapter_metadata_b"
```

Comment:
@chandrasekharan-zipstack  Will this cause  a change in doc_id and hence require re-indexing as we discussed the other day?

Reply:
No @gaya3-zipstack , I made sure to pop these extra columns out to not break any existing code and just log them. If they're required elsewhere we could add an extra function / perform a migration